app/models/practice.rb

```diff
@@ -0,0 +1,7 @@
+class Practice < ActiveRecord::Base
+  belongs_to :user
+  has_many   :daily_progresses
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.